### ISSUE TITLE
Add connection status notification icon

### ISSUE THREAD

**Issue by IndrekV** [toggl/toggldesktop#1739]
Should add to dropdown menu or preferences. (Needs to be decided)

Green dot working ok
Red dot not working

In there show two statuses:

Websocket ok/down (websocket connection is up or not) 
Api connection ok/down (ok - last sync was success, down - last sync failed/there are unsynced items)
